data/releases/schedule.yaml

```diff
@@ -67,11 +76,14 @@ schedules:
       targetDate: 2020-12-18
       note: "Tagging Issue https://groups.google.com/g/kubernetes-dev/c/dNH2yknlCBA"
 - release: 1.19
-  next: 1.19.15
-  cherryPickDeadline: 2021-09-10
-  targetDate: 2021-09-15
+  next: 1.19.16
+  cherryPickDeadline: 2021-10-15
+  targetDate: 2021-10-20
   endOfLifeDate: 2021-10-28
```

Comment:
should we update the EOL to be the day of this Oct patch release? or we might make a final release in November?

Reply:
I'd agree with @puerco. Unless we have something critical, we should adhere to the defined guidelines and not release another patch release. In that case, I don't think it makes sense to add it to the schedule.

Reply:
agree as well, will update the PR, thanks for the feedback!